Doc/library/contextlib.rst

```diff
@@ -137,6 +137,23 @@ Functions and classes provided:
    ``page.close()`` will be called when the :keyword:`with` block is exited.
 
```

Comment:
Declare an explicit hyperlink anchor here with a blank line before the function definition:

    .. _simplifying-support-for-single-optional-context-managers:

    .. function:: nullcontext(enter_result=None)